docs/administering/backups.md

```diff
@@ -12,7 +12,17 @@ server.
 
 If you want to get crafty, you can modify a backup to point to a different app ID, allowing you to
 migrate data between apps, or use your data with an experimental app with a different app ID. You
-can also change the contents of the backup before restoring it.
+can also change the contents of the backup before restoring it. To ensure Sandstorm accepts the modified backup please follow these steps:
+- Backup the grain and download the zip file
+- unpack the zip file
+- to edit the app id
+  - create a new dummy Sandstorm package (see https://github.com/sandstorm-io/vagrant-spk)
+  - copy the new app id from the packages `sandstorm-pkgdef.capnp` file
+  - open the `metadata` file from the unpacked zip foder and exchange the app id in that file
```

Comment:
This is a binary file. What are you opening it with? A text editor will probably corrupt it.